crates/modelardb_server/src/storage/uncompressed_data_buffer.rs

```diff
@@ -36,6 +36,10 @@ use tracing::debug;
 
 use crate::storage::{StorageEngine, UNCOMPRESSED_DATA_BUFFER_CAPACITY, UNCOMPRESSED_DATA_FOLDER};
 
+/// Number of [`RecordBatches`](RecordBatch) that must be ingested without modifying an
+/// [`UncompressedInMemoryDataBuffer `] before it is considered unused and can be finished.
```

Comment:
It seems like there is an extra space in "[`UncompressedInMemoryDataBuffer `]"

Reply:
Fixed.